### PR TITLE
New MedTech service template with an Azure IoT Hub

### PR DESCRIPTION
**Pull request by msjasteppe**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [X] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*  New MedTech service quickstart template with an Azure IoT Hub.

### CODE REVIEW

**Comment by azure-quickstarts** [comment]
@msjasteppe - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@msjasteppe - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@msjasteppe - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@msjasteppe - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@msjasteppe - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@msjasteppe - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@msjasteppe - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by msjasteppe** [comment]
@bmoore-msft My PR seems to be stuck at the deployment phase of the AzQuickStarts-MAC test. Is there something wrong with our template that is causing this issue? Adding @namalu for visibility.

**Comment by azure-quickstarts** [comment]
@msjasteppe - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@msjasteppe - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by msjasteppe** [comment]
#sign-off

**Comment by msjasteppe** [comment]
#sign-off

File: quickstarts/microsoft.healthcareapis/workspaces/iotconnectors-with-iothub/main.bicep [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+  'westus3'
+  'uksouth'
+])
+param location string
+
+@description('OPTIONAL - A principal id for a user that will be granted FHIR Data Contributor access to the FHIR service. If you do not choose to use the **fhirContributorPrincipalId** option, clear the field of any entries. To learn more about how to acquire an Azure AD user object ID, see [Find the user object ID](https://learn.microsoft.com/partner-center/find-ids-and-domain-names#find-the-user-object-id)')
+param fhirContributorPrincipalId string = ''
+
+@description('The mapping JSON that determines how incoming device data is normalized.')
+param deviceMapping object = {
```
Comment (bmoore-msft) [inline quickstarts/microsoft.healthcareapis/workspaces/iotconnectors-with-iothub/main.bicep:30]: have you tried a portal deployment with defaultValues this large?

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: quickstarts/microsoft.healthcareapis/workspaces/iotconnectors-with-iothub/main.bicep (hunk 1 above)
Comment (msjasteppe) [reply quickstarts/microsoft.healthcareapis/workspaces/iotconnectors-with-iothub/main.bicep:30]: @bmoore-msft 

We have and it has been successful in creating the mappings (device and destination).

image

image

**Pull request merged**